### PR TITLE
STP-3667: Tech pubs clean-up for `docs-sat` move to `docs-csm`

### PR DESCRIPTION
**Pull request by jld-dehn-hpe**
## Description

Tech pubs is reviewing the work that moved `docs-sat` content into `docs-csm` for the CSM 1.6 release. This is part of tech pubs' handoff of the SAT product documentation to engineering going forward. 

The initial work to move `docs-sat` into `docs-csm` was done by Ryan Haasken in PR #4476. That work is described in more detail as part of Ryan's [Confluence page](https://rndwiki-pro.its.hpecorp.net/display/CASMSAT/2023-11-15+SAT+full+inclusion+in+CSM+v1.6.0 ). This PR includes some clean-up work as described in [STP-3667](https://jira-pro.it.hpe.com:8443/browse/STP-3667).

- Update "CNE" references to "USS" since the product was renamed before release
- Remove references to SAT as an independent product with independent documentation
- Clean-up work to pass repo checks and improve wording

## Checklist

- [x] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [x] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

 
[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

File: introduction/documentation_conventions.md [hunk 1]
```diff
@@ -6,20 +6,20 @@ This outlines conventions and standards that are used in this documentation.
...
+        - [`kubectl exec` example](#kubectl-exec-example)
+        - [Combined `ssh` and `chroot` example](#combined-ssh-and-chroot-example)
+        - [`sat bash` example](#sat-bash-example)
+    - [Command prompts in Python virtual environments](#command-prompts-in-python-virtual-environments)
+    - [Directory path in command prompt](#directory-path-in-command-prompt)
 - [Ability to pause and resume procedures](#ability-to-pause-and-resume-procedures)
-  - [Bad example](#bad-example)
-  - [Good example](#good-example)
+    - [Bad example](#bad-example)
+    - [Good example](#good-example)
```
Comment (haasken-hpe) [inline introduction/documentation_conventions.md:22]: Can you explain the rationale behind these space changes?

File: workflows/ncn/hooks/before-each/cos-prechecks-for-worker-reboots.yaml [hunk 2]
```diff
@@ -38,8 +38,8 @@ spec:
     ssh ${SSH_OPT} ${TARGET_NCN} "\
       if [[ -f /opt/cray/shasta/uss/bin/prechecks_for_worker_reboots ]]; then \
           /opt/cray/shasta/uss/bin/prechecks_for_worker_reboots; \
-      elif [[ -f /opt/cray/shasta/cne/bin/prechecks_for_worker_reboots ]]; then \
-          /opt/cray/shasta/cne/bin/prechecks_for_worker_reboots; \
+      elif [[ -f /opt/cray/shasta/uss/bin/prechecks_for_worker_reboots ]]; then \
+          /opt/cray/shasta/uss/bin/prechecks_for_worker_reboots; \
```
Comment (haasken-hpe) [inline workflows/ncn/hooks/before-each/cos-prechecks-for-worker-reboots.yaml]: It looks like this script is checking for the existence of a file named `prechecks_for_worker_reboots` in three different placed while trying to account for the various naming iterations of the COS/CNE/USS product, and then it's executing the one that's found. The lines above (39-40) already check for the existence of the script with `uss` in the path, so this one should be left to fall back on the `cne` location, if it exists. So I'd revert the changes to this file.

**Review by haasken-hpe — CHANGES_REQUESTED** [review]
I think you should revert the change to `workflows/ncn/hooks/before-each/cos-prechecks-for-worker-reboots.yaml`

File: introduction/documentation_conventions.md (hunk 1 above)
Comment (jld-dehn-hpe) [reply introduction/documentation_conventions.md:22]: @haasken-hpe, the markdown style check wouldn't pass without these changes. I think it's trying to enforce line 24 [here ](https://github.com/Cray-HPE/docs-csm/blob/release/1.6/.editorconfig) that indentations be 4 spaces.

File: workflows/ncn/hooks/before-each/cos-prechecks-for-worker-reboots.yaml (hunk 2 above)
Comment (jld-dehn-hpe) [reply workflows/ncn/hooks/before-each/cos-prechecks-for-worker-reboots.yaml]: Reverting changes in next commit

File: introduction/documentation_conventions.md (hunk 1 above)
Comment (haasken-hpe) [reply introduction/documentation_conventions.md:22]: I could be wrong, but I think it's actually [here](https://github.com/Cray-HPE/docs-csm/blob/release/1.6/.github/config/markdown_style.yaml#L59) that controls the markdown style checker.

Either way, it does look like both locations were changed by https://github.com/Cray-HPE/docs-csm/pull/4719 and that occurred after my change to this list, so it makes sense why it didn't flag it for me, but did for you.

**Review by haasken-hpe — APPROVED** [review]
Looks good. Thanks for cleaning up after me and making these updates!

**Review by annapoorna-s-alt — APPROVED** [review]

**Pull request merged**